install-daemon.sh

```diff
@@ -113,6 +113,12 @@ function detect_distro {
 }
 
 function check_os_comp {
+  MACHINE_TYPE=$(uname -m)
+  if [ "${MACHINE_TYPE}" != "x86_64" ]; then # check the architecture
```

Comment:
Does Pterodactyl (or any of the dependencies) only support 64-bit?

Reply:
No, it may support different architectures, but that wasn't my point. The docker images ,which egg creators use, are mostly based on 64 bit (in my experience). And the commands inside the docker containers also may not be supported by other architectures. But adding a bypass for those who want to take the risk would solve this.

Reply:
Very good point, closing conversation.

Reply:
Should I still make a bypass?